ui/src/app/settings/actions/config.js

```diff
@@ -4,6 +4,7 @@ config.fetch = () => {
   return {
     type: "FETCH_CONFIG",
     meta: {
+      model: "config",
       method: "config.list",
```

Comment:
I wonder if the method should just be "list" and we combine the model and method in `buildMessage`. Not sure.